functions/common/consts/maxrfise-error-codes.ts

```diff
@@ -0,0 +1,16 @@
+import { StatusCodes } from "../types";
+
+export const MaxfriseErrorCodes = {
```

Comment:
This is just mid term solution while we sort out this in the external library for error codes